ts/private/build_test.bzl

```diff
@@ -0,0 +1,35 @@
+"""Fork of @bazel_skylib//rules:build_test.bzl
+
+Workaround for https://github.com/bazelbuild/bazel-skylib/issues/480
+"""
+
+_DOC = """\
+build_test is a trivial test that always passes.
+
+It is used to force Bazel to build some of its dependencies, in the case where
+`--build_tests_only` would have otherwise caused those to be skipped.
+"""
+
+def _build_test_impl(ctx):
+    executable = ctx.actions.declare_file(ctx.label.name + ".sh")
+    ctx.actions.write(
+        output = executable,
+        is_executable = True,
+        content = "#!/usr/bin/env bash\nexit 0",
+    )
+
+    return DefaultInfo(
+        files = depset([executable]),
+        executable = executable,
+        # The critical fix, so that the test inputs include the default outputs of the targets to check
```

Comment:
> so that the test **outputs** include
?

Reply:
no, it's an input to the test. We want the test to be a cache miss when that input changes.

Reply:
I think there's just too many inputs + outputs and this doesn't clarify which. This is the "test action" inputs? Not the `build_test` inputs?